authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/config/ComputeKeyHelperConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.mosip.authentication.common.service.config;
+
```

Comment:
Please create one config file called BaseConfig and make all the beans in one file

Reply:
Done